agent/handlers/v2/task_container_metadata_handler.go

```diff
@@ -31,6 +31,9 @@ const (
 	// TaskMetadataPath specifies the relative URI path for serving task metadata.
 	TaskMetadataPath = "/v2/metadata"
 
+	// TaskWithTagsMetadataPath specifies the relative URI path for serving task metadata with Container Instance and Task Tags.
+	TaskWithTagsMetadataPath = "/v2/metadataWithTags"
```

Comment:
Just to confirm, did you discuss this with akram?

Reply:
Brought it up and he deferred to our team. v2 and v3 metadata responses are built the same way. To be consistent, it is better to provide the Tags in both responses.